### ISSUE TITLE
database not updating after audio files are converted to mp3

### ISSUE THREAD

**Issue by tsk-2222** [axcore/tartube#451]
Windows 10
Tartube v2.4.093

I have a folder for audio only files. These videos are downloaded as an m4a, converted into mp3, and then the original m4a is deleted. So far so good.

The problem is that the database does not get updated and it is still looking for the m4a file and I have to manually go into properties for each one and pick the mp3 file.

This will also cause a database duplicate where I now have an m4a and an mp3 item.

**Comment by axcore** [comment]
Download and convert a single video as normal, then click the **Output** tab and show me the yellow text at the top, so I can try to reproduce it.

**Comment by tsk-2222** [comment]
```
yt-dlp --newline -i --max-filesize 9g -r 800K --hls-prefer-native --write-description --write-info-json --cookies F:/TSK/Media/!Youtube/cookies.txt --write-thumbnail --sleep-interval 2 --max-sleep-interval 10 -f m4a -x --audio-format mp3 --embed-thumbnail --add-metadata --windows-filenames --force-overwrites --sleep-requests 1 --sleep-subtitles 10 --check-formats --remux-video mp3 --convert-thumbnails jpg --output F:/TSK/Media/!Youtube/Audio/%(uploader)s_%(upload_date>%Y-%m-%d)s_%(title)s.%(ext)s --download-archive F:/TSK/Media/!Youtube/Audio/ytdl-archive.txt https://www.youtube.com/watch?v=djV11Xbc914
[youtube] djV11Xbc914: Downloading webpage
[youtube] Sleeping 1.0 seconds ...
[youtube] djV11Xbc914: Downloading android player API JSON
[info] Testing format 140
[info] djV11Xbc914: Downloading 1 format(s): 140
[info] Downloading video thumbnail 41 ...
[info] Writing video description to: F:/TSK/Media/!Youtube/Audio/a-ha_2010-01-06_a-ha - Take On Me (Official Video) [Remastered in 4K].description
[info] Writing video thumbnail 41 to: F:/TSK/Media/!Youtube/Audio/a-ha_2010-01-06_a-ha - Take On Me (Official Video) [Remastered in 4K].webp
[info] Writing video metadata as JSON to: F:/TSK/Media/!Youtube/Audio/a-ha_2010-01-06_a-ha - Take On Me (Official Video) [Remastered in 4K].info.json
[ThumbnailsConvertor] Converting thumbnail "F:/TSK/Media/!Youtube/Audio/a-ha_2010-01-06_a-ha - Take On Me (Official Video) [Remastered in 4K].webp" to jpg
Deleting original file F:/TSK/Media/!Youtube/Audio/a-ha_2010-01-06_a-ha - Take On Me (Official Video) [Remastered in 4K].webp (pass -k to keep)
[download] Sleeping 5.46 seconds ...
[download] Destination: F:/TSK/Media/!Youtube/Audio/a-ha_2010-01-06_a-ha - Take On Me (Official Video) [Remastered in 4K].m4a
[download] 100% of 3.77MiB in 00:04 at 789.77KiB/s
[ExtractAudio] Destination: F:/TSK/Media/!Youtube/Audio/a-ha_2010-01-06_a-ha - Take On Me (Official Video) [Remastered in 4K].mp3
Deleting original file F:/TSK/Media/!Youtube/Audio/a-ha_2010-01-06_a-ha - Take On Me (Official Video) [Remastered in 4K].m4a (pass -k to keep)
[Metadata] Adding metadata to "F:/TSK/Media/!Youtube/Audio/a-ha_2010-01-06_a-ha - Take On Me (Official Video) [Remastered in 4K].mp3"
[VideoRemuxer] Not remuxing media file "F:/TSK/Media/!Youtube/Audio/a-ha_2010-01-06_a-ha - Take On Me (Official Video) [Remastered in 4K].mp3"; already is in target format mp3
[EmbedThumbnail] ffmpeg: Adding thumbnail to "F:/TSK/Media/!Youtube/Audio/a-ha_2010-01-06_a-ha - Take On Me (Official Video) [Remastered in 4K].mp3"
```

It seems everything happens as it should, except when finished, the database is pointing to a non-existent m4a file.  I could now go into the file properties and make the file point to the mp3.  With that manual repair step, everything is as it should be.

Doing a folder refresh before the manual repair just makes things even worse by causing duplicates and some other strangeness.

**Comment by axcore** [comment]
Fixed in v2.6.165. Old downloads won't magically update themselves, but new downloads should work exactly as expected.
